src/shared/ui/layout/ContentWithPreview.tsx

```
@@ -58,14 +58,14 @@ export function ContentWithPreview({
       </div>
       <div
         className={cn(
-          // Use the shared glass-panel visual treatment but remove rounded corners
-          // so the preview content can fill the full panel area while keeping
-          // the backdrop / glass appearance present.
-          'glass-panel rounded-none p-0 min-h-0 overflow-y-auto border-t border-line-glass/30 lg:border-l lg:border-t-0',
+          // Preview panel with subtle background and proper padding
+          'bg-slate-50 min-h-0 overflow-y-auto border-l border-line-glass/40',
```

ESLint

[error] Prefer system tokens (surface-*, input-*, or accent-*) over hardcoded colors to ensure proper theme inversion.

(custom/no-hardcoded-colors)